analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -446,15 +446,16 @@ class MavenSupport(private val workspaceReader: WorkspaceReader) {
 
             if (resultForPomFile != null) {
                 log.warn {
-                    "There was an error building project '${e.projectId}' at '${e.pomFile.invariantSeparatorsPath}'. " +
+                    "There was an error building project '${e.projectId}' at " +
+                            "'${e.pomFile?.invariantSeparatorsPath}'. " +
                             "Still continuing with the incompletely built project '${resultForPomFile.projectId}' at " +
-                            "'${resultForPomFile.pomFile.invariantSeparatorsPath}': ${e.collectMessages()}"
+                            "'${resultForPomFile.pomFile?.invariantSeparatorsPath}': ${e.collectMessages()}"
                 }
 
                 resultForPomFile
             } else {
                 log.error {
-                    "Failed to build project '${e.projectId}' at '${e.pomFile.invariantSeparatorsPath}': " +
+                    "Failed to build project '${e.projectId}' at '${e.pomFile?.invariantSeparatorsPath}': " +
```

Comment:
Could / should we do something nicer and maybe say something like "unknown file" instead of "null"? Because in case of "null" the user does not know what entity "null" refers to.